assets/js/components/Dashboard/Dashboard.tsx

```diff
@@ -40,9 +41,27 @@ const Dashboard = (props: { page: string }): JSX.Element => {
   const [screenTypeFilterValue, setScreenTypeFilterValue] = useState(
     SCREEN_TYPES[0]
   );
+  const [showScreenlessPlaces, setShowScreenlessPlaces] = useState(true);
   const [statusFilterValue, setStatusFilterValue] = useState(STATUSES[0]);
   const [activeEventKeys, setActiveEventKeys] = useState<string[]>([]);
 
+  const resetFilters = () => {
+    setModeLineFilterValue(MODES_AND_LINES[0]);
+    setScreenTypeFilterValue(SCREEN_TYPES[0]);
+    setStatusFilterValue(STATUSES[0]);
+    setShowScreenlessPlaces(true);
+  };
+
+  const goToHome = () => {
+    // This approach is difficult to maintain--every time we add new state values,
+    // we need to remember to add a line here to "reset" it.
+    // TODO: Should the home button simply be a link to /dashboard? That would reload the page,
```

Comment:
Ahh, that sounds like exactly what we want. I know when the button is clicked, all state should be wiped out. I believe that will include filters on the Alerts page as well. If that link will handle all of that, that would be great.